ui/ui-components/client/client.ts

```diff
@@ -124,8 +125,6 @@ export class Client {
       data: null,
       url: `${this.webUrl}/api/${this.apiVersion}${path}`, // path comes with a leading "/"
       withCredentials: true,
-      //@ts-ignore
```

Comment:
I guess the `//@ts-ignore` should have been a hint this wasn't quite right :D

Reply:
What does _typescript_ know 😂